ansible_ai_connect/users/pipeline.py

```diff
@@ -124,69 +120,3 @@ def redhat_organization(backend, user, response, *args, **kwargs):
         "rh_user_is_org_admin": user.rh_user_is_org_admin,
         "external_username": user.external_username,
     }
-
-
-def _terms_of_service(strategy, user, backend, **kwargs):
-    accepted = "terms_accepted"
-    is_commercial = user.rh_user_has_seat
-    if not settings.ANSIBLE_AI_ENABLE_TECH_PREVIEW:
-        return {accepted: True}
-    # Commercial & local users are not presented with T&C page in login flow (new & existing users)
-    if settings.TERMS_NOT_APPLICABLE or is_commercial:
-        return {accepted: True}
-
-    field_name = "community_terms_accepted"
-    view_name = "community_terms"
-    terms_accepted = strategy.session_get(accepted, None)
-    if getattr(user, field_name, None) is not None:
-        # User had previously accepted, so short-circuit the T&C page.
-        return {accepted: True}
-
-    if terms_accepted is None:
-        # We haven't gone through the flow yet -- go to the T&C page
-        current_partial = kwargs.get("current_partial")
-        return strategy.redirect(f"{reverse(view_name)}?partial_token={current_partial.token}")
-
-    if not terms_accepted:
-        raise AuthCanceled("Terms and conditions were not accepted.")
-
-    # We've accepted the T&C, set the field on the user.
-    setattr(user, field_name, timezone.now())
-    user.save()
-    return {accepted: terms_accepted}
-
-
-@partial
-def terms_of_service(strategy, details, backend, user=None, is_new=False, *args, **kwargs):
-    return _terms_of_service(strategy, user, backend, **kwargs)
-
-
-class AuthAlreadyLoggedIn(AuthException):
-    def __str__(self):
-        return "User already logged in"
-
-
-def block_auth_users(backend=None, details=None, response=None, user=None, *args, **kwargs):
-    """Safeguard to be sure user won't get multiple providers"""
-    if user:
-        raise AuthAlreadyLoggedIn(backend)
-
-
-def load_extra_data(backend, details, response, uid, user, *args, **kwargs):
-    social = kwargs.get("social") or backend.strategy.storage.user.get_social_auth(
-        backend.name, uid
-    )
-    if social:
-        extra_data = backend.extra_data(user, uid, response, details, *args, **kwargs)
-        user.external_username = extra_data.get("login")
-        user.save()
-        social.extra_data["aap_licensed"] = (
```

Comment:
We need these properties for AAP user mapping. I don't see them elsewhere in this PR. 

Without them we loose certain [functionality](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/users/models.py#L132-L144) for "on prem" including protection of the `/metrics` [endpoint](https://github.com/ansible/ansible-ai-connect-service/blob/main/ansible_ai_connect/main/views.py#L127-L145).